.github/workflows/release.yml

```diff
@@ -1,11 +1,76 @@
-name: release
+name: Terraform Provider Release Deployer
 on:
-  push:
-    tags:
-      - 'v*'
+  workflow_dispatch:
+    inputs:
+      release-version:
```

Comment:
If you have to specify the `release-type`, I don't think we require the `release-version`, right?

Reply:
You are right. `release-type` itself is sufficient but that makes the process a bit blind as you won't be able to check exactly which version is being deployed. `release-version` is added to make the release deployer more aware of the version he/she is deploying.

But anyways, if you want. I can make the necessary modifications.

Reply:
You're right, I'm ok to go with what you did :)
Maybe we can also ask @elodietinland since she was the first one to work on terraform if it makes sense for here

Reply:
Ok it makes sense to me. Just to be sure : the merge of a PR doesn't trigger the workflow ? To trigger it, you need to do it manually like it's explained in the README ?

Reply:
Yup. The merge of a PR won't trigger the workflow. As mentioned in the README, you have to manually run it. :slightly_smiling_face: